development-guidelines.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+---
+
+# Development Guidelines
+
```

Comment:
Sure, I can add that. I was wondering whether we have general guidelines or just setup a page/document per project that are applicable and specific to it?